tools/docker/fabric-all-in-one/Dockerfile

```diff
@@ -34,6 +34,14 @@ RUN apk add --no-cache curl
 # The file binary is used to inspect exectubles when debugging container image issues
 RUN apk add --no-cache file
 
+# Download and setup path variables for Go
+RUN wget https://golang.org/dl/go1.13.4.linux-amd64.tar.gz
```

Comment:
Any reason why not using the latest stable go version? I think it was 1.15.5. If Fabric needs an older one for whatever reason that's fine then.

Reply:
Incorporated the changes. Please have a re-look.